docs/guide/build_hardware.md

```diff
@@ -107,7 +107,9 @@ Plug in the Servo driver the same as the Raspberry Pi, just keep in mind that th
 
 ![Jetson Servo](/assets/Servo_Wiring.png)
 
-Finally this is the Donkey Assembled.  
+Finally this is the Donkey assembled.  
+
+Although it may seem reasonable to connect all pins in the set of six from the servo, the second pin is *OE Output enable. ... When the pin is high the outputs are disabled. Pulled low by default so it's an optional pin!* Connecting this pin can result in your car unexpectedly stopping.
```

Comment:
Maybe move it just below the line 106?

Reply:
there's only 3 pins from the servo. I don't get it.

Reply:
http://docs.donkeycar.com/assets/Servo_Wiring.png

It's these close six pins I was referring to. I guess I meant "servo driver". Unless there's a clearer term to use..?

Reply:
@BrianHenryIE you mean 6 pins on PCA controller (there are cables connected to it 0 black, gray, white, violet)?

Reply:
@nvtkaszpir Yes, the six pins on the PCA controller (GND, OE, SCL, SDA, VCC, V+). If OE is connected the car will shut off unexpectedly. The kit I bought from [DonkeyCar.com](https://store.donkeycar.com/) had a set of five joined wires/connectors so I just connected them all. 

@tawnkramer It's the pins on the PCA9865 that connect to my Jetson Nano I'm referring to.  The photo isn't mine but displays things properly.